mona_openai/endpoints/completion.py

```diff
@@ -0,0 +1,43 @@
+"""
+The Mona wrapping code for OpenAI's Completion API.
+"""
+from copy import deepcopy
+
+from ..analysis.analysis import get_full_analysis
+from ..util.validation_util import validate_openai_class
+
+COMPLETION_CLASS_NAME = "Completion"
+
+
+def get_completion_class(openai_class):
+    """
+    Returns a monitored class wrapping the given openai class, which is
+    expected to be a "openai.Completion" class, enriching it with
+    specific capabilities to be used by an inhereting monitored class.
+    """
+    validate_openai_class(openai_class, COMPLETION_CLASS_NAME)
+
+    class MonitoredCompletion(openai_class):
+        @classmethod
+        def _get_analysis_params(cls, input, response, specs):
+            return get_full_analysis(
+                input["prompt"],
```

Comment:
I think named params here will help understand better what is the purpose of each one of them

Reply:
I actually think this one is pretty straight forward, but if you feel that way that's a good enough reason to change.